presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/ObjectIdFunctions.java

```diff
@@ -63,6 +66,14 @@ public static Slice ObjectId(@SqlType(StandardTypes.VARCHAR) Slice value)
         return Slices.wrappedBuffer(new ObjectId(CharMatcher.is(' ').removeFrom(value.toStringUtf8())).toByteArray());
     }
 
+    @ScalarFunction("get_timestamp")
```

Comment:
Nit: The name here is redundant, since the default name is the snake case version of the camel case Java name.